app_server/app/adapters/socket/Client.js

```diff
@@ -17,7 +17,7 @@ function Client(socket, credentials) {
 
   this.socket = socket;
   this.credentials = credentials;
-  this.rooms = {};
+  this.rooms = {}; // A map from room's name to Room object
```

Comment:
Is there a reason why there are 2 room mappings in `Client.js` and `RoomsManager.js`